api/managementpb/checks.proto

```diff
@@ -79,7 +80,7 @@ service SecurityChecks {
       body: "*"
     };
   }
-  // ListSecurityChecks returns a list of available Security Thread Tool checks and their statuses.
+  // ListSecurityChecks returns a list of available Security Thread Tool checks, their statuses and description.
```

Comment:
For me, "check" there means "static check fields". Statuses were added explicitly because they are runtime parameters, not static fields.

The current reading implies that check is only a name, and the description is somewhat a separate thing, which is not true.

Let's either remove the description from the comment or remove both description and status.